internal/provider/resource_access_generic.go

```diff
@@ -140,27 +155,290 @@ func (a *genericJAASAccessResource) Configure(ctx context.Context, req resource.
 		)
 		return
 	}
-	a.client = client
+	resource.client = client
 	// Create the local logging subsystem here, using the TF context when creating it.
-	a.subCtx = tflog.NewSubsystem(ctx, a.resourceLogName)
+	resource.subCtx = tflog.NewSubsystem(ctx, resource.resourceLogName)
 }
 
 // Create defines how tuples for access control will be created.
-func (a *genericJAASAccessResource) Create(ctx context.Context, req resource.CreateRequest, resp *resource.CreateResponse) {
+func (resource *genericJAASAccessResource) Create(ctx context.Context, req resource.CreateRequest, resp *resource.CreateResponse) {
+	// Check first if the client is configured
+	if resource.client == nil {
+		addClientNotConfiguredError(&resp.Diagnostics, resource.resourceLogName, "create")
+		return
+	}
+
+	// Read Terraform configuration from the request into the model
+	plan, targetTag := resource.info(ctx, req.Plan, &resp.Diagnostics)
+	if resp.Diagnostics.HasError() {
+		return
+	}
 
+	// Create tuples to create from the plan
+	tuples := modelToTuples(ctx, targetTag, plan, &resp.Diagnostics)
+	if resp.Diagnostics.HasError() {
+		return
+	}
+	// Make a call to create relations
+	err := resource.client.Jaas.AddRelations(tuples)
+	if err != nil {
+		resp.Diagnostics.AddError("Client Error", fmt.Sprintf("Unable to create access relationships, got error: %s", err))
+		return
+	}
+	// Set the plan onto the Terraform state
+	resp.Diagnostics.Append(resource.targetResource.Save(ctx, &resp.State, plan, targetTag)...)
 }
 
 // Read defines how tuples for access control will be read.
-func (a *genericJAASAccessResource) Read(ctx context.Context, req resource.ReadRequest, resp *resource.ReadResponse) {
+func (resource *genericJAASAccessResource) Read(ctx context.Context, req resource.ReadRequest, resp *resource.ReadResponse) {
+	// Check first if the client is configured
+	if resource.client == nil {
+		addClientNotConfiguredError(&resp.Diagnostics, resource.resourceLogName, "read")
+		return
+	}
+	// Read Terraform configuration from the request into the plan
```

Comment:
todo: update comment, you're reading from the request into the resource model, not the plan.